utils/graph_sampler.jl

```diff
@@ -1,19 +1,30 @@
 using ABCDGraphGenerator
+using Random
 
-@info "Usage: julia graph_sampler.jl networkfile communityfile degreefile communitysizesfile μ isCL islocal"
-@info "Example: julia graph_sampler.jl network.dat community.dat degrees.dat community_sizes.dat 0.2 true true"
+@info "Usage: julia graph_sampler.jl networkfile communityfile degreefile communitysizesfile mu|xi fraction isCL islocal [seed]"
```

Comment:
seed processing seems to be missing here

Reply:
I'm not sure what is missing - `seed` is specified in Usage, Example and is processed in line 23 below

Reply:
I missed 23